pageserver/src/bin/pageserver.rs

```diff
@@ -383,7 +383,7 @@ fn start_pageserver(
     let shutdown_pageserver = tokio_util::sync::CancellationToken::new();
 
     // Set up remote storage client
-    let remote_storage = create_remote_storage_client(conf)?;
+    let remote_storage = Some(create_remote_storage_client(conf)?);
```

Comment:
This is probably unrelated change from another PR: https://github.com/neondatabase/neon/pull/7722/files